packages/providers/inpage-providers-hub/src/floatingButton/index.tsx

```
@@ -186,54 +198,59 @@
+              transition: 'all 150ms cubic-bezier(0.4, 0, 0.2, 1)',
+              opacity: showCloseButton ? 1 : 0,
+              borderRadius: '9999px',
+              backgroundColor: '#fff',
+              border: '1px solid rgba(0, 0, 0, 0.1)',
+            }}
+            onClick={(event) => {
+              event.stopPropagation();
+              setIsShowCloseButton(false);
+              showCloseDialog();
+            }}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)